### PR TITLE
update rayon version

### PR DESCRIPTION
**Pull request by ljiang-lyft**
Upgrade rayon to unblock using upgraded s3b version https://github.com/lyft/avmapping-geometry/pull/725

Tested with xray_viewer of avmapping-geometry
`./xray_viewer --log-level info --port 9000`
image

### CODE REVIEW

**Comment by mcristina422** [comment]
To loosen the requirements a bit, also consider using https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html instead of hardcoding to one specific version

**Comment by ljiang-lyft** [comment]
> To loosen the requirements a bit, also consider using https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html instead of hardcoding to one specific version

@mcristina422  Yeah the trickly part is how to test. I'm not too familiar with this repo, lemme do some more reading on the testing section on both avmapping_geometry and this repo. Will continue in an hour.

File: xray/Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ nalgebra = "0.22.0"
 num = "0.3.0"
 pbr = "1.0.3"
 protobuf = "2.18.0"
-rayon = "1.4.1"
+rayon = "1.5.1"
```
Comment (feuerste) [inline xray/Cargo.toml:26]: I recommend to update the rayon version in all `Cargo.toml` files, so we are consistent throughout the repository

Comment (ljiang-lyft) [reply xray/Cargo.toml:26]: Done. Interesting, this time's change does not generate new changes in cargo.lock

**Comment by ljiang-lyft** [comment]
@feuerste it looks i'm not allowed to retrggier the ci run. I closed and reopend this PR, it does not got retriggered as well. Could you please help trigger?

**Comment by ljiang-lyft** [comment]
> > @feuerste it looks i'm not allowed to retrggier the ci run. I closed and reopend this PR, it does not got retriggered as well. Could you please help trigger?
> 
> @ljiang-lyft You need to click on `Details` next to the `DCO` error and follow the instructions first.

HMM this is all I got
image
I already followed the instruction to run it for the first 7 commits, now it was asking for doing the same thing for 15 commits ...
What does this mean? Can I just set. it to pass? @feuerste

**Comment by ljiang-lyft** [comment]
@feuerste Can you just simple manually set it to `pass` for the dco test?  
Also is the CI test still up-to-date, I'm seeing the same error when running ci locally on latest master branch. The suggested changes looks not too useful to me, WDYT?
image

**Comment by ljiang-lyft** [comment]
Replacing this PR with https://github.com/cartographer-project/point_cloud_viewer/pull/481 which does not have the unncessary changes to address the CI warning errrros.

**Pull request closed without merging**